roboticarm:
  hardware_interface:
     loop_hz: 10 # hz
     joints:
       - j0
       - j1
       - j2
       - j3
       - j4